### PR TITLE
#170672722 - Live Notifications

### PR DESCRIPTION
**Pull request by nignanthomas**
#### What does this PR do?
Live Notifications.

#### Description of Task to be completed?
- Use socket to emit on notification.

#### How should this be manually tested?
- Clone the repository from [here](https://github.com/andela/caret-bn-backend.git)
- Browse to the repo directory
- Navigate to ch-live-notifications-170672722branch
- Run `npm install`
- Run `sequelize db:migrate`
- Run `sequelize db:seed:all`
- Run `npm run dev`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170672722](https://www.pivotaltracker.com/story/show/170672722)

#### Questions:
N/A

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -58,6 +58,9 @@
         socket.on("connection_error", message => {
           console.log(message);
         });
+        socket.on("notification", notifData => {
+          console.log('notification ===> ', notifData);
```
Comment (alainmateso) [inline src/index.html]: remove this console.log

File: src/index.html [hunk 2]
```diff
@@ -58,6 +58,9 @@
...
+        });
         socket.on("disconnect", err => {
           console.log("Server disconnected, please reload!");
```
Comment (alainmateso) [inline src/index.html]: remove this console.log

File: src/index.js [hunk 3]
```diff
@@ -50,6 +52,31 @@ io.on('connect', socket => {
...
+  try {
+    const userData = decodeToken(token);
+    if (userData) {
+      const clientKey = Number.parseInt(userData.payload.id, 10);
+      connectedClients[clientKey] = connectedClients[clientKey] || [];
+      connectedClients[clientKey].push(socket.id);
+    }
+    next();
+  } catch (error) {
+    console.log('error ===> ', error);
```
Comment (alainmateso) [inline src/index.js]: remove this console.log

**Review by alainmateso — APPROVED** [review]
Good Job @nignanthomas , kindly remove a few console.log I have mentioned

**Review by ngireric123 — APPROVED** [review]
LGTM

**Pull request merged**